src/sw.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference lib="webworker" />
+import {
+  cleanupOutdatedCaches,
+  createHandlerBoundToURL,
+  precacheAndRoute,
+} from "workbox-precaching";
+import { NavigationRoute, registerRoute } from "workbox-routing";
+import { activatePolyfills } from "./web-features";
+
+// TODO: review import.meta issue
+// import { activatePolyfills } from "@web5/api";
```

Comment:
@csuwildcat to cause the error that I mentioned and troubleshoot why the service worker is crashing you can just switch imports to use the one from the `@web5/api` (uncommenting the bottom part).

then `pnpm build` and `http-server ./dist`, `open http://localhost:8080`, and inspect the service worker console